### PR TITLE
Fix problem with winrm

### PR DESCRIPTION
**Pull request by rukawata**
There are some issues with the previous winrm configuration, that makes the ansible not able to run playbook on windows jumpboxes, especially with the problem below:

_Not able to logon and run playbook on windows jumpbox during the 1st terraform+ansible deployment - we see ansible hanging at [**Gathering facts**] phase. However the 2nd attempt of terraform + ansible would work._

This is the fix based on [the PR for python3](https://github.com/Azure/sap-hana/pull/280), and have tested with [sample playbook](https://github.com/Azure/sap-hana/tree/nancyc-winrm-fix-test) that it works from end to end for windows jumpbox.

### CODE REVIEW

File: deploy/v2/terraform/modules/jumpbox/main.tf [hunk 1]
```diff
@@ -285,9 +294,9 @@ resource "null_resource" "prepare-rti" {
       # Install pip3
       "sudo apt -y install python3-pip",
       # Installs Ansible
-      "sudo apt install software-properties-common",
-      "sudo apt-add-repository --yes --update ppa:ansible/ansible-2.8",
-      "sudo apt -y install ansible=2.8*",
+      "sudo -H pip3 install \"ansible>=2.8,<2.9\"",
```
Comment (tniek) [inline deploy/v2/terraform/modules/jumpbox/main.tf]: Why ansible < 2.9?

File: deploy/v2/terraform/modules/jumpbox/outputs.tf [hunk 2]
```diff
@@ -22,3 +22,7 @@ output "rti-info" {
 output "prepare-rti" {
   value = null_resource.prepare-rti
 }
+
+output "vm-windows" {
+  value = azurerm_virtual_machine.vm-windows
+}
```
Comment (tniek) [inline deploy/v2/terraform/modules/jumpbox/outputs.tf]: Probably needs to be addressed in a separate PR, but not happy about the inconsistent naming (`prepare-rti` vs. `vm-windows`)

File: deploy/v2/terraform/modules/jumpbox/main.tf (hunk 1 above)
Comment (rukawata) [reply deploy/v2/terraform/modules/jumpbox/main.tf]: We want to fix using ansible larger release 2.8.
This was decided in https://github.com/Azure/sap-hana/pull/266.
If we would move to 2.9.* at certain point, we could change it by then.

Comment (rukawata) [reply deploy/v2/terraform/modules/jumpbox/main.tf]: > Why ansible < 2.9?

We want to fix using ansible larger release 2.8.
This was decided in #266.
If we would move to 2.9.* at certain point, we could change it by then.

File: deploy/v2/terraform/modules/jumpbox/outputs.tf (hunk 2 above)
Comment (rukawata) [reply deploy/v2/terraform/modules/jumpbox/outputs.tf]: > Probably needs to be addressed in a separate PR, but not happy about the inconsistent naming (`prepare-rti` vs. `vm-windows`)

No. This is an workaround (same as `output "prepare-rti"`) and is required to make sure windows jumpboxes is ready before ansible-playbook runs.

Comment (rukawata) [reply deploy/v2/terraform/modules/jumpbox/outputs.tf]: > Probably needs to be addressed in a separate PR, but not happy about the inconsistent naming (`prepare-rti` vs. `vm-windows`)

For the inconsistent naming, I will address that in a different PR since the names were already defined in the previous PRs.
